packages/sdk/src/modules/FusePools.ts

```diff
@@ -77,6 +77,7 @@ export function withFusePools<TBase extends MidasBaseConstructor>(Base: TBase) {
 
         if (_asset) {
           asset.underlyingSymbol = _asset.symbol;
+          asset.logoUrl = "https://d1912tcoux65lj.cloudfront.net/token/96x96/" + _asset.symbol.toLowerCase() + ".png";
```

Comment:
What is this one about? A bit confused why we set this. Only thing which comes to mind is the strategy logo but we have that already o.O?

Reply:
I think we could set the token info in the SDK directly rather than in UI because we need to call hook `useTokenData` in UI to only get token image